backend/internal/database/database.go

```diff
@@ -0,0 +1,62 @@
+package database
+
+import (
+	"github.com/ibeckermayer/teleport-interview/backend/internal/model"
+	"github.com/jmoiron/sqlx"
+	"github.com/pborman/uuid"
+)
+
+// Config is a database config object
+type Config struct {
+	env string
+}
+
+// NewConfig returns a new Config for the database
+func NewConfig(env string) Config {
+	return Config{env}
+}
```

Comment:
nit: might be simpler to just export the `env` field in `Config` and get rid of the constructor